### PR TITLE
dev-core-102-14

### PR DESCRIPTION
**Pull request by deflillusion**
Список дел, за основу взят код из backend-101

### CODE REVIEW

File: dev-core-102/lesson-14/app/main.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                    status_input = input(
+                        "Введите новый статус (0 — не выполнена, 1 — выполнена): ").strip()
+                    new_status = status_input == "1"
+
+                result = update_task(task_id, new_title or None, new_status)
+                print(result["message"])
+            except ValueError:
+                print("Ошибка: ID задачи или статус должны быть числом.")
+            except HTTPException as e:
+                print(e.detail)
```
Comment (aissayev) [inline dev-core-102/lesson-14/app/main.py:212]: for a cleaner code, I would suggest to break down each option code handling to a separate method
```
def handle_choice_4()
  try:
     ....



```

**Review by aissayev — APPROVED** [review]
lgtm, one comment to make code even more cleaner

File: dev-core-102/lesson-14/app/main.py (hunk 1 above)
Comment (deflillusion) [reply dev-core-102/lesson-14/app/main.py]: Исправил. Вывел в отдельную функцию

Comment (aissayev) [reply dev-core-102/lesson-14/app/main.py]: same applies for other choices - like 1, 2, 3

**Pull request merged**